docs/release_notes/v1.11.1.md

```diff
@@ -0,0 +1,113 @@
+# Dapr 1.11.1
+
+## Fixed an issue with Service Invocation on Kubernetes clusters with the permission enforcement admission controller enabled.
+
+### Problem
+
+Updating Dapr from 1.10 to 1.11 on Kubernetes clusters gives the following error
+when using Service Invocation:
+
+```
+unable to create Dapr service for wrapper, service: test/test-dapr, err:
+services \"test-dapr\" is forbidden: cannot set blockOwnerDeletion if an
+ownerReference refers to a resource you can't set finalizers on:
+```
+
+### Impact
+
+Since v1.11.0 Service Invocation on a subset of Kubernetes clusters is unusable.
```

Comment:
```suggestion
Since v1.11.0 Service Invocation on Kubernetes clusters which have the permission enforcement admission controller enabled is unusable.

This is particularly problematic when for OpenShift clusters, as they have this admission controller enabled by default.
```